### PR TITLE
dc invite spec

### PR DESCRIPTION
**Pull request by Simon-Laux**
standard/spec draft for https://github.com/deltachat/invite

### CODE REVIEW

File: dc-invite.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+todo why?
+
+todo what?
+
+
+## How?
+
+Deltachat will register to the following domains
+- `https://invite.delta.chat/*` - official instace hosted by us 
+- `https://dcinvite.*` - subdomain on any server for people that would like to selfhost it
```
[the remark this answers is not in the document]
Comment (Simon-Laux) [reply dc-invite.md:12]: not sure. I'm not even sure that this attempt at documentation is still up to date. anyways we should document it here.

[the remark this answers is not in the document]
Comment (r10s) [reply dc-invite.md:12]: iirc, it is possible on android, not sure about ios. but yes, it seems better to drop that selfhosting part completely for now and make the bare thing work

**Comment by r10s** [comment]
i suggest to closing this stale PR.

there is no need to write a spec if the things is not really shipped yet; it also gets easily outdated and not sure if there is a need at all

**Comment by Simon-Laux** [comment]
> let's better put resources in finishing invite.delta.chat and not on the spec the the first place.

We should have a spec in the end that describes the thing as documentation on how the process works, also this file contains ideas not yet implemented (registering for a custom domain / deep linking to the apps).

**Pull request closed without merging**